README.md

```diff
@@ -71,73 +71,56 @@ The `--fix` option on the command line automatically fixes problems reported by
 
 <!--RULES_TABLE_START-->
```

Comment:
the sentence above "All rules below with a check mark :white_check_mark: are enabled by default while using `plugn:ember/base` or `plugin:ember/recommended` configs." needs to be adjusted